Doc/howto/logging-cookbook.rst

```diff
@@ -2732,10 +2732,10 @@ governs the formatting of logging messages for final output to logs, and is
 completely orthogonal to how an individual logging message is constructed.
 
 Logging calls (:meth:`~Logger.debug`, :meth:`~Logger.info` etc.) only take
-positional parameters for the actual logging message itself, with keyword
-parameters used only for determining options for how to handle the logging call
-(e.g. the ``exc_info`` keyword parameter to indicate that traceback information
-should be logged, or the ``extra`` keyword parameter to indicate additional
+positional arguments for the actual logging message itself, with keyword
```

Comment:
ditto
```suggestion
positional (or keyword) arguments for the logging message itself, with keyword
```
This paragraph is pretty redundant with the on 1100 lines above, but I guess this would be a separate issue.

Reply:
```suggestion
positional arguments for the logging message itself, with keyword
```